orcid-core/src/main/resources/orcid-core-context.xml

```diff
@@ -741,131 +741,7 @@
 	<bean id="passwordGenerationManager"
 		class="org.orcid.core.manager.impl.PasswordGenerationManagerImpl">
 		<constructor-arg value="12"></constructor-arg>
-	</bean>		
-
-    <alias name="incomingValidationManagerV1_2_rc6" alias="incomingValidationManagerLatest"/>
-    <alias name="outgoingValidationManagerV1_2_rc6" alias="outgoingValidationManagerLatest"/>
-    
-    <bean id="incomingValidationManagerRoot" class="org.orcid.core.manager.impl.ValidationManagerForLegacyApiVersionsImpl">
```

Comment:
Hey @auumgn please also check those java classes. I think the following ones can be removed as well: 

- org.orcid.core.manager.impl.ValidationManagerForLegacyApiVersionsImpl
- org.orcid.core.manager.impl.ValidationManagerImpl